libs/backend/src/cast_vote_records/canonicalize.ts

```diff
@@ -20,47 +16,6 @@ const EMPTY_PAGE_TYPES: ReadonlyArray<PageInterpretation['type']> = [
   'UnreadablePage',
 ];
 
-/**
- * Enumeration representing possible sheet validation errors
- */
-export enum SheetValidationErrorType {
-  InvalidFrontBackPageTypes = 'InvalidFrontBackPageTypes',
-  MismatchedBallotStyle = 'MismatchedBallotStyle',
-  MismatchedBallotType = 'MismatchedBallotType',
-  MismatchedElectionHash = 'MismatchedElectionHash',
-  MismatchedPrecinct = 'MismatchedPrecinct',
-  NonConsecutivePages = 'NonConsecutivePages',
-}
-
-/**
- * Type containing a validation error and information relevant to that error
- */
-export type SheetValidationError =
```

Comment:
Had to move this type to `libs/types`, since I later need it for a `userReadableMessageFromExportError` helper in `libs/ui`